Maman11/Q1/my_add.c

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define BYTE_SIZE 8
+
+void print_binary(int num) {
```

Comment:
Add code documentation for all functions.
Inputs, outputs, how does the function work

this is an example:
/*
 This function finds suffix in a string that start with a given char
 When the function finds the suffix it prints it and counts it


 The function iterates over the string using a for loop.

 param str: (char[]) the string to iterate over
 param c: (char) the char that start the suffix.

 @return: (int) the number of suffix in the string.
          if a suffix is found the program prints it.
*/

I was lazy with the explanation but this is the format